sdk/communication/communication-sms/test/smsClientWithToken.spec.ts

```diff
@@ -1,51 +1,66 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { record, Recorder, env } from "@azure/test-utils-recorder";
-import { SendRequest, SmsClient } from "../src/smsClient";
+import { env, isPlaybackMode, record, Recorder } from "@azure/test-utils-recorder";
+import { SmsSendRequest, SmsClient } from "../src/smsClient";
 import { assert } from "chai";
 import { isNode } from "@azure/core-http";
 import * as dotenv from "dotenv";
+import * as sinon from "sinon";
 import { parseConnectionString } from "@azure/communication-common";
 import { createCredential, recorderConfiguration } from "./utils/recordedClient";
+import { Uuid } from "../src/utils/uuid";
 
 if (isNode) {
   dotenv.config();
 }
 
-describe("SmsClientWithToken [Playback/Live]", async () => {
-  let recorder: Recorder;
+const isBrowser = new Function("try {return this===window;}catch(e){ return false;}");
```

Comment:
Well, `isBrowser` is as simple as `!isNode` which we've already imported.
But I don't think we should use it to turn off tests.